chain/indexer/README.md

```diff
@@ -0,0 +1,67 @@
+# NEAR Indexer
+
+NEAR Indexer is a micro-framework, which provides you with a stream of blocks that are recorded on NEAR network.
+
+
+NEAR Indexer is useful to handle real-time "events" on the chain.
+
+
+NEAR Indexer is going to be used to build NEAR Explorer, augment NEAR Wallet, and provide overview of events in Rainbow Bridge.
+
+
+See the [example](https://github.com/nearprotocol/nearcore/tree/master/tools/indexer/example) for further details.
+
+
+## How to set up and test NEAR Indexer
+
+Before you proceed, make sure you have the following software installed:
+* [rustup](https://rustup.rs/) or Rust version that is mentioned in `rust-toolchain` file in the root of nearcore project.
+
+### localnet
+
+Clone [nearcore](https://github.com/nearprotocol/nearcore)
+
+```bash
+$ git clone git@github.com:nearprotocol/nearcore.git
+$ cd nearcore
+$ cargo run --package neard --bin neard init
```

Comment:
I suggest we add `--release` here to avoid double compilation given that we recommend compiling Indexer Example with `--release`.

```suggestion
$ env "NEAR_HOME=$HOME/.near/localnet" cargo run --release --package neard --bin neard init
```